src/ucp/wireup/select.c

```diff
@@ -270,7 +270,7 @@ ucp_wireup_select_transport(const ucp_wireup_select_params_t *select_params,
     p            = tls_info;
     endp         = tls_info + sizeof(tls_info) - 1;
     tls_info[0]  = '\0';
-    tl_bitmap   &= select_params->tl_bitmap;
+    tl_bitmap   &= (select_params->tl_bitmap & context->tl_bitmap);
```

Comment:
unrelated fix?

Reply:
yes, added here because PR is small and simple

Reply:
logically it's different unrelated change so should be in different PR
what is it fixing? which test is failing?

Reply:
actually, it's not a fix, micro optimization which just reduces extra iterations when upper func passes tl_bitmap=UINT64_MAX